### PR TITLE
feat(adapter): use with firefox

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: modules/RTC/RTCBrowserType.js [hunk 1]
```diff
@@ -164,8 +164,9 @@ const RTCBrowserType = {
      * @returns {boolean}
      */
     usesNewGumFlow() {
-        return RTCBrowserType.isChrome()
-            && RTCBrowserType.getChromeVersion() >= 61;
+        return (RTCBrowserType.isChrome()
+            && RTCBrowserType.getChromeVersion() >= 61)
+            || RTCBrowserType.isFirefox();
```
Comment (bbaldino) [inline modules/RTC/RTCBrowserType.js]: let's double check if we can move all versions of firefox over.

File: modules/xmpp/SDPUtil.js [hunk 2]
```diff
@@ -588,7 +588,10 @@ const SDPUtil = {
         }
         if (payloadType) {
             const payloadTypes
-                = videoMLine.payloads.split(' ').map(p => parseInt(p, 10));
+                = videoMLine.payloads
+                    .toString()
```
Comment (bbaldino) [inline modules/xmpp/SDPUtil.js]: i'd put a comment about the issue you found here that requires the `toString` call

File: modules/RTC/RTCBrowserType.js (hunk 1 above)
Comment (saghul) [reply modules/RTC/RTCBrowserType.js]: Drive by comment: do we have a policy on what versions of Firefox we support? I guess that supporting >= ESR should be ok, that is, >= 52. Older versions could be considered unsupported.

Comment (bbaldino) [reply modules/RTC/RTCBrowserType.js]: not that i know of, but having a policy of something like `N versions behind stable` would definitely be nice (and make this sort of work much more straightforward)

**Comment by bbaldino** [comment]
jenkins, test this please

File: modules/RTC/RTCBrowserType.js (hunk 1 above)
Comment (virtuacoplenny) [reply modules/RTC/RTCBrowserType.js]: I uploaded this branch to my instance, used Firefox 50 and a quick sanity check passed, including screensharing with the extension. The version displayed in ESR 52.4.1.

**Comment by saghul** [comment]
Great work, less code -> more better! :-)

**Pull request merged**